### PR TITLE
Data interface boilerplate file

### LINKED ISSUE #12: Create DatabaseInterface
## Story
Create a file to manage access to the database

### PR DESCRIPTION
**Pull request by DreRandaci**
## Link to Ticket
Closes #12  [Create Database Interface](https://github.com/spooky-oysters/Bangazon-Command-Line-Application/issues/12)

## Description of Proposed Changes

- Added a `DatabaseInterface.cs` file to manage access to the `BANGAZON_CLI_APP_DB.db`

## Steps to Test

```sh
git fetch --all
git checkout dr-DbInterface
```
- Check that there is a `Data/DatabaseInterface.cs` file and that the code comments are sufficient 
- Add an environment variable path to your `.bash_profile` or `.zshrc` file named `BANGAZON_CLI_APP_DB`
- - Example: 
```sh
export BANGAZON_CLI_APP_DB="/Users/You/workspace/csharp/Bangazon-Command-Line-Application/dataFileNameYouWantToUse.db"
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

* This will allow each developer methods to access their local database file to perform all CRUD operations

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documentation with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.

### CODE REVIEW

**Comment by kimberly-bird** [comment]
👍

**Comment by Chewieez** [comment]
I think line 30 should not include the word `Environment`. This is a little confusing since this isn't an environment variable, it's going to hold the contents of an environment variable.

Look at line 67 and reword

I would reword line 103 to: `// Loop runs once and assigns the inserted items ID to the initialized insertedItemId variable`

I would move the `If it throws an exception, create the table` from line 130 to 134 or 135, right where the code for the exception is. It's confusing to read it in the code for the success.

**Comment by Chewieez** [comment]
👍  good jorb
